packages/create-webstone-app/template/plugin/extension.ts

```diff
@@ -0,0 +1,7 @@
+import type { GluegunToolbox } from "@webstone/gluegun";
+
+export default (toolbox: GluegunToolbox) => {
```

Comment:
Created a dummy extension, not sure what to put in the dummy file that gets generated here

Reply:
That looks fine. Another idea is "Please edit path-to-this-file to extend the CLI toolbox"